anodet/feature_extraction.py

```diff
@@ -4,7 +4,7 @@
 
 import torch
 import torch.nn.functional as F
-from torchvision import models
+from torchvision.models import resnet18, ResNet18_Weights, wide_resnet50_2, Wide_ResNet50_2_Weights
```

Comment:
Did you mean to include this in the pr? I saw that you had another commit with these changes. If you bother I think it's nicer with two different pr's, to keep them simple.